test: add integration tests for put

tmyoda opened the pull request:
*Issue #, if available:*
#143 

*Description of changes:*
This pull request adds integration tests for put command.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

StoneDot reviewed (commented):
Thank you for implementing great tests for the `put` command. To make this pull request more excellent one, could you please consider adding the following tests?

* Check the results of multiple `put` invocations for different primary keys.
* Check the results of multiple `put` invocations for the same primary key.

Since this pull request has the required quality to be merged, I have approved it. If you add any additional tests, please let me know. I will review the changes.

tmyoda commented on tests/put.rs:
```diff
@@ -0,0 +1,442 @@
+        }
+    }]
+    "#;
+    for aciton in ["put", "p"] {
```
I have updated here to verify the command alias "p"

tmyoda commented on tests/put.rs:
```diff
@@ -0,0 +1,442 @@
+        let mut c = tm.command()?;
+        let cmd = c.args([
+            "--region",
+            "local",
+            "--table",
+            &table_name,
+            "put",
+            "42",
+            &sk.to_string(),
+        ]);
+        cmd.assert().success().stdout(format!(
+            "Successfully put an item to the table '{}'.\n",
+            table_name
+        ));
+
+        items.push(json!({
+            "pk": {
+                "S": "42"
+            },
+            "sk": {
+                "S":sk.to_string()
+            }
+        }));
+    }
+
+    let mut c = tm.command()?;
+    let get_cmd = c.args([
+        "--region",
+        "local",
+        "--table",
+        &table_name,
+        "scan",
+        "-o",
+        "raw",
+    ]);
+
+    let expected = json!(items).to_string();
+    util::assert_eq_json_ignore_order(get_cmd, &expected);
+    Ok(())
+}
```
I have added 3 test based on your review.
- test_multiple_put_different_pk
- test_multiple_put_same_pk
- test_multiple_put_same_pk_different_sk

tmyoda commented:
I have added 5 tests based on your review and internal meetings:

From internal meetings.
-  `test_put` (added a test for the command alias "p")

Check the results of multiple put invocations for different primary keys.
- `test_multiple_put_different_pk`

Check the results of multiple put invocations for the same primary key.
-  `test_multiple_put_same_pk`

I have also added these tests to make sure that put replaces the existing item and for different sort key:
-  `test_put_same_pk`
-  `test_multiple_put_same_pk_different_sk`

StoneDot commented on tests/put.rs:
```diff
@@ -0,0 +1,487 @@
+        .create_temporary_table_with_items(
+            "pk",
+            None,
+            vec![util::TemporaryItem::new(
```
Actually, we no longer do not need to use vec!. We can just pass the array. However, we can modify it later in other pull request.

StoneDot replied:
I think this should be handled in another issue based on current usage of the method in other tests.

StoneDot reviewed (approved):
Thank you for polishing up this pull request. LGTM
